content/pages/constitution.md

```diff
@@ -112,7 +114,9 @@ computing.
     4. To track the completion of all duties assigned to members of the
        committee;
 
-    5. To maintain the non-financial records of the committee.
+    5. To manage all non-computing-related volunteers;
```

Comment:
Yeah, I guess I was imagining non-Systems Committee volunteers that are nonetheless working on computer stuff. But I think syscom technically covers all of that. Will change.